crates/swc_css_lints/src/rules/no_duplicate_at_import_rules.rs

```diff
@@ -0,0 +1,78 @@
+use std::fmt::Display;
+
+use ahash::AHashSet;
```

Comment:
Please use `AHashSet` from `swc_common` instead.

Reply:
(For consisstency.)

Reply:
`ahash::AHashSet` and `ahash::AHashMap` are newtype structs, while those in `swc_common` is type alias for standard hashmap type.